src/hooks/Filters/product-context.jsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useReducer,useContext } from "react";
+import { useProducts } from "../useProduct";
```

Comment:
I can see you have made an index.jsx file but not used it for barrel importing here.

Reply:
@GuptaEesh can you check the code once... I worked on the suggestions.